CONTRIBUTING.md

```diff
@@ -25,12 +25,62 @@ In particular, this community seeks the following types of contributions:
 * A developer of the [open-build-service team](https://github.com/orgs/openSUSE/teams/open-build-service) will review your pull-request
   * If the pull request gets a positive review the reviewer will merge it
 
-We tag our commits depending on the area that is affected by the change. Those are
-* [webui]   - Changes in webui related parts of app/model/ and lib/ as well as app/controllers/webui/ and it's views
+
+## How to write proper commit messages
+
+### Tag your commits
+
+We tag our commits depending on the area that is affected by the change. **All commits should start with at least one tag** from:
+
 * [api]     - Changes in api related parts of app/model/ and lib/ as well as app/controllers/\*.rb and it's views
 * [backend] - Changes in the perl-written backend of OBS
 * [ci]      - Changes that affect our test suite
+* [dist]    - Modifies something inside /dist directory
 * [doc]     - Any documentation related changes
+* [webui]   - Changes in webui related parts of app/model/ and lib/ as well as app/controllers/webui/ and it's views
+
+In case of having more than one tag, they should be **alphabetically** ordered.
+
+
+### Useful commit subject
```

Comment:
Sorry not google, github -> https://github.com/erlang/otp/wiki/Writing-good-commit-messages